app/models/tagset_models/tag_set.rb

```diff
@@ -4,10 +4,11 @@ class TagSet < ApplicationRecord
   # a complete match is numerically represented with ALL
   ALL = -1
 
-  TAG_TYPES = %w(fandom character relationship freeform category rating warning)
+  TAG_TYPES = %w(fandom character relationship freeform category rating archive_warning)
```

Comment:
Digging up an old comment from @tickinginstant that pertains to this file, although I'm not sure if it's still necessary: https://github.com/otwcode/otwarchive/pull/3526#issuecomment-478804944

The line in question is:
https://github.com/otwcode/otwarchive/blob/ab0697a2b42c839e5dcb484d03ce50caa0ba1932/app/models/tagset_models/tag_set.rb#L222

And the suggestion was to make that `underscore` instead of `downcase`.